Servicelayer/GetUserSer/GetUserSer.go

```diff
@@ -1,21 +0,0 @@
-package GetUserSer
-
-import (
-	"example/persons/models"
-	"strconv"
-	"github.com/gin-gonic/gin"
-	"example/persons/repository/GetUserRepo"
-)
-func GetUserSer(c *gin.Context)([]models.Person,error) {
```

Comment:
try not to cut words.
suggestion: GetUser

Reply:
done